<ui version="4.0" >
 <class>newMapForm</class>
 <widget class="QWidget" name="newMapForm" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>281</width>
    <height>151</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Create new map</string>
  </property>
  <widget class="QSpinBox" name="widthBox" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>30</y>
     <width>62</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimum" >
    <number>1</number>
   </property>
   <property name="value" >
    <number>30</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="tilesetDisplay" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>80</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="tilesetButton" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>110</y>
     <width>261</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text" >
    <string>Browse</string>
   </property>
  </widget>
  <widget class="QLabel" name="label" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>60</y>
     <width>62</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text" >
    <string>TileSet:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>10</y>
     <width>62</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text" >
    <string>Width:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3" >
   <property name="geometry" >
    <rect>
     <x>90</x>
     <y>10</y>
     <width>62</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text" >
    <string>Height:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="heightBox" >
   <property name="geometry" >
    <rect>
     <x>90</x>
     <y>30</y>
     <width>62</width>
     <height>30</height>
    </rect>
   </property>
   <property name="value" >
    <number>30</number>
   </property>
  </widget>
  <widget class="QPushButton" name="createButton" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="geometry" >
    <rect>
     <x>177</x>
     <y>30</y>
     <width>91</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text" >
    <string>Create</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
